src/query/ast/src/parser/statement.rs

```diff
@@ -2063,7 +2070,8 @@ pub fn statement_body(i: Input) -> IResult<Statement> {
             | #show_locks : "`SHOW LOCKS [IN ACCOUNT] [WHERE ...]`"
             | #kill_stmt : "`KILL (QUERY | CONNECTION) <object_id>`"
             | #vacuum_temp_files : "VACUUM TEMPORARY FILES [RETAIN number SECONDS|DAYS] [LIMIT number]"
-            | #set_priority: "SET PRIORITY (HIGH | MEDIUM | LOW) <object_id>"
+            | #set_priority: "`SET PRIORITY (HIGH | MEDIUM | LOW) <object_id>`"
+            | #set_backtrace: "`(ENABLE | DISABLE) EXCEPTION_BACKTRACE`"
```

Comment:
I strongly oppose the idea of introducing two new keywords for this feature. Can we reuse an existing one like `SET`? For example:

```sql
SET EXCEPTION_BACKTRACE TRUE
SET EXCEPTION_BACKTRACE FALSE
```

Reply:
I found @zhang2014 said that `SET` is used for tenant. I am not sure if it lead to misunderstanding.

Reply:
I checked our [document](https://docs.databend.com/sql/sql-commands/administration-cmds/set-global). Could we add `GLOBAL` parameter to `SET` command to express this 
semantic. e.g.`SET GLOBAL EXCEPTION_BACKTRACE=TRUE`

Reply:
We can introduce the system statement.

Reply:
> We can introduce the system statement(SYSTEM ENABLE BACKTRACE).

How will user use it in sql? Does it look like the following:

```sql
system enable EXCEPTION_BACKTRACE
```

Reply:
Do we really need `ENABLE` and `DISABLE`? Can we reuse `SET` so users can:

```rust
system set EXCEPTION_BACKTRACE=true
```

Please note `EXCEPTION_BACKTRACE` is not a keyward too, just like `SET max_threads = 4;`

Reply:
I feel `system set EXCEPTION_BACKTRACE=true` is more reasonable. What do you think? @zhang2014

Reply:
Enable disable will be more explicit.